Documentation/ceph-block.md

```diff
@@ -187,19 +187,91 @@ Continue with the example above for the [wordpress application](#consume-the-sto
 
 ## Advanced Example: Erasure Coded Block Storage
 
-> **IMPORTANT**: This is only possible when using the Flex driver. Ceph CSI 1.2 (with Rook 1.1) does not support this type of configuration yet.
-
 If you want to use erasure coded pool with RBD, your OSDs must use `bluestore` as their `storeType`.
 Additionally the nodes that are going to mount the erasure coded RBD block storage must have Linux kernel >= `4.11`.
 
-To be able to use an erasure coded pool you need to create two pools (as seen below in the definitions): one erasure coded and one replicated.
-The replicated pool must be specified as the `blockPool` parameter. It is used for the metadata of the RBD images.
-The erasure coded pool must be set as the `dataBlockPool` parameter below. It is used for the data of the RBD images.
+**NOTE**: This example requires *at least 3 bluestore OSDs*, with each OSD located on a *different node*.
+
+The OSDs must be located on different nodes, because the [`failureDomain`](ceph-pool-crd.md#spec) is set to `host` and the `erasureCoded` chunk settings require at least 3 different OSDs (2 `dataChunks` + 1 `codingChunks`).
 
+To be able to use an erasure coded pool you need to create two pools (as seen below in the definitions): one erasure coded and one replicated.
 > **NOTE**: This example requires *at least 3 bluestore OSDs*, with each OSD located on a *different node*.
 
 The OSDs must be located on different nodes, because the [`failureDomain`](ceph-pool-crd.md#spec) is set to `host` and the `erasureCoded` chunk settings require at least 3 different OSDs (2 `dataChunks` + 1 `codingChunks`).
 
+### EC CSI Driver Example
+
+The erasure coded pool must be set as the `dataPool` parameter below. It is used for the data of the RBD images.
+
+```yaml
```

Comment:
Just like mentioned in https://github.com/rook/rook/pull/4726/files#r369424410 I'd like to remove this example from the doc and only refer to the yaml from the repo.